engine/backends/proverif/proverif_backend.ml

```diff
@@ -115,35 +115,207 @@ module Print = struct
   let library_functions :
       (Concrete_ident_generated.name * (AST.expr list -> document)) list =
     [
-      (Core__ops__try_trait__Try__branch, fun args -> empty);
-      (* just an example *)
+      (* Core dependencies *)
```

Comment:
What do you plan on doing with those?
Do you plan to rename each of those manually to something else?

Since you are building the library on ProVerif's side as well, can't you just name functions with a scheme we can directly produce, without the indirection of rewriting all those symbols?

Reply:
Well, I mean that instead of manually saying `hax_lib_protocol::cal::hmac` is actually `foobar` in the proverif side, why not calling that `cal_hmac` in proverif, and just translate all the item `hax_lib_protocol::cal::foo` as `cal_foo`? or even, why not use a fully qualified name there? i.e. `hax_lib_protocol__cal__hmac`

Reply:
Ah, I think I understand now, thanks! I'm wondering if that would only work if we just want translate one for one to matching names and not to something more complicated on the ProVerif side.
An example may be when we want to drop some of the arguments of a function when translating to ProVerif, then we would need special logic for that translation in particular. Sorry, this is a bit abstract right now. Do you think I could address it properly when I work on #447 and #448, where I have to fill in the translations?